consensus/hotstuff/verification/single_verifier.go

```diff
@@ -65,7 +64,6 @@ func (s *SingleVerifier) VerifyQC(voterIDs []flow.Identifier, sigData []byte, bl
 	if len(signers) < len(voterIDs) { // check we have valid consensus member Identities for all signers
 		return false, fmt.Errorf("some signers are not valid consensus participants at block %x: %w", block.BlockID, model.ErrInvalidSigner)
```

Comment:
I really liked that you also included the case of ID duplication in your error message for [`CombinedVerifier`](https://github.com/onflow/flow-go/blob/541e3acd5aac8e69b62bf7ebc88fdeb682107a1c/consensus/hotstuff/verification/combined_verifier.go)! Lets do the same here

```suggestion
		return false, fmt.Errorf("duplicate or invalid signer IDs in QC for block %x: %w", block.BlockID, model.ErrInvalidSigner)
```